x-pack/plugins/observability_solution/observability_onboarding/server/routes/logs/route.ts

```diff
@@ -30,6 +31,16 @@ const logMonitoringPrivilegesRoute = createObservabilityOnboardingServerRoute({
   },
 });
 
+function createScriptDownloadUrl(staticAssets: IStaticAssets, kibanaHost: string) {
+  const scriptPath = staticAssets.getPluginAssetHref('standalone_agent_setup.sh');
+  const isAbsolute = scriptPath.startsWith('http');
+
+  if (isAbsolute) {
+    return scriptPath;
+  }
+  return `${kibanaHost}${scriptPath}`;
+}
```

Comment:
I would use Web APIs where possible since they are more reliable than bespoke logic:

```
return new URL(scriptPath, kibanaHost).toString();
```

See: https://developer.mozilla.org/en-US/docs/Web/API/URL/URL

Reply:
Oh, nice, I did not know about that contructor with `base`, only tried the ` new URL(url)` but that one throws for relative paths. Thank you for the tip!